components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -119,6 +143,13 @@ export default function TeamSettings() {
                     >
                         Update Organization
                     </Button>
+
+                    <CheckBox
```

Comment:
The current UX leaves an awkward loading state where we render an unchecked box until we've loaded team settings, then it might change w/o the user doing anything. Maybe as a small improvement we can make the checkbox disabled while we're loading team settings so the user doesn't try to change it before we've loaded settings.

```ts
// can extract an isLoading prop from the `useOrgSettingsQuery()` result above
disabled={orgSettingsQuery.isLoading}
```